content/working_with/official_plugins/Orchestration/helm.md

```diff
@@ -183,33 +160,15 @@ Helm plugin uses `curl` on  `installation_source` and unzip it, then move it to
 
 {{< highlight  yaml  >}}
 
-inputs:
-
-  helm_executable:
-    description: >
-      Helm binary path.
-    type: string
-    default: '/tmp/helm'
-
-  helm_installation_source:
-    description: >
-      Helm download link.
-    type: string
-    default: 'https://get.helm.sh/helm-v3.3.1-linux-amd64.tar.gz'
-
-dsl_definitions:
-
-  helm_config: &helm_config
-    executable_path: { get_input: helm_executable }
-
 node_templates:
 
   helm_install:
     type: cloudify.nodes.helm.Binary
     properties:
-      helm_config: *helm_config
+      helm_config:
+        executable_path: '/tmp/helm'
       use_existing_resource: false
-      installation_source: { get_input: helm_installation_source }
+      installation_source: 'https://get.helm.sh/helm-v3.3.1-linux-amd64.tar.gz'
```

Comment:
Perhaps add that this is an example in the docs and we cannot guarantee that this URL is permanent.

Reply:
OK